File: AulaAprendaJavaDoZero/app/src/main/java/com/application/aulaaprendajavadozero/MainActivity.java
package com.application.aulaaprendajavadozero;

import androidx.appcompat.app.AppCompatActivity;
import android.os.Bundle;

import com.application.aulaaprendajavadozero.classes.ContaBancaria;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        ContaBancaria conta = new ContaBancaria();

    }
}

File: AulaInjecaoDependenciasHilt/app/src/main/java/com/application/aulainjecaodependenciashilt/MainActivity.kt
package com.application.aulainjecaodependenciashilt

import android.os.Bundle
import androidx.appcompat.app.AppCompatActivity
import com.application.aulainjecaodependenciashilt.classes.Banda
import com.application.aulainjecaodependenciashilt.classes.Carro
import com.application.aulainjecaodependenciashilt.classes.Musico
import dagger.hilt.android.AndroidEntryPoint
import javax.inject.Inject

@AndroidEntryPoint
class MainActivity : AppCompatActivity() {

    //@Inject lateinit var carro: Carro
    //@Inject lateinit var musico: Musico
    @Inject lateinit var banda: Banda

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        //carro.ligar()
        //musico.tocar()
        banda.tudoSendoTocado()
    }
}